### ISSUE TITLE
Inaccurate installation instructions for Conan

### ISSUE THREAD

**Issue by tomaszkalisiak-digica** [Dobiasd/frugally-deep#353]
The `INSTALL.md` tutorial provides the following bit of configuration as a guide for Conan users:

    [requires]
    frugally-deep/v0.15.18-p0@dobiasd/stable
        
However, this will not work because version `v0.15.18-p0` of the package has not been uploaded to Conan Center. Attempting to use that dependency specification will yield a message similar to the following:

    ERROR: Failed requirement 'frugally-deep/v0.15.18-p0@dobiasd/stable' from 'conanfile.py (test/0.0.1)'
    ERROR: Unable to find 'frugally-deep/v0.15.18-p0@dobiasd/stable' in remotes

In short - I would like to request uploading the latest version of the package to Conan Center. If it's impossible, please consider modifying the installation guide. Perhaps showing installation for Conan directly from the Github repository, like for Hunter, if it's possible at all.

**Comment by Dobiasd** [comment]
Oh, thanks for reminding me. :grimacing:

I tend to forget to update the package on Conan when releasing a new version. (same as ~ [half a year ago](https://github.com/Dobiasd/frugally-deep/issues/313)) Somehow I always incorrectly expect Conan to be able to automatically pick up the new version, as other package managers already do. :shrug:

I'll make `v0.15.18-p0` available on Conan and let you know when done. :office_worker:

**Comment by tomaszkalisiak-digica** [comment]
Thank you for responding so quickly, I really appreciate it. : )

**Comment by Dobiasd** [comment]
FYI, my PR was merged a few minutes ago. So the installation should start working now. :-)
